MerlinAU.sh

```diff
@@ -6351,19 +6377,47 @@ _WAN_IsConnected_()
    return "$retCode"
 }
 
-##----------------------------------------##
-## Modified by Martinski W. [2024-Nov-13] ##
-##----------------------------------------##
+##------------------------------------------##
+## Modified by ExtremeFiretop [2024-Nov-15] ##
+##------------------------------------------##
 _PostUpdateEmailNotification_()
 {
    _DelPostUpdateEmailNotifyScriptHook_
+   FW_UpdateCheckState="$(nvram get firmware_check_enable)"
+   [ -z "$FW_UpdateCheckState" ] && FW_UpdateCheckState=0
+   if [ "$FW_UpdateCheckState" -eq 1 ] && [ "$sendEMailNotificationsFlag" ]
+   then
+       # Check if the CRON job already exists #
+       if ! eval $cronListCmd | grep -qE "$CRON_JOB_RUN #${CRON_JOB_TAG}#$"
+       then
+           if _AddCronJobEntry_
+           then
+               printf "Cron job '${GRNct}${CRON_JOB_TAG}${NOct}' was added successfully.\n"
+               _AddCronJobRunScriptHook_
+           else
+              printf "${REDct}**ERROR**${NOct}: Failed to add the cron job [${CRON_JOB_TAG}].\n"
+           fi
+       else
+           printf "Cron job '${GRNct}${CRON_JOB_TAG}${NOct}' already exists.\n"
+           _AddCronJobRunScriptHook_
+       fi
```

Comment:
My expected solution for this PR. This addresses your points 1 and 2.
However - TO BE REMOVED? (If and When) you come up with a better solution.

Reply:
> My expected solution for this PR. This addresses your points 1 and 2. However - TO BE REMOVED? (If and When) you come up with a better solution.
>
>
It's no longer needed to delete/add the cron job hook after the latest changes in the "`_AcquireLock_`" function are merged.

Reply:
Done!